_posts/2016-01-20-Making-of-Peacock.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Making of Peacock
+excerpt: Behind the scenes of making Swagger support in Apiary.
+layout: post
+date: 2016-01-20 00:00:00 +0100
+author: zdenek
+published: true
+---
+
+Recently, we have launched the biggest feature in the history of Apiary: [Swagger Support](http://blog.apiary.io/2016/01/18/We-ve-got-Swagger). This project was nicknamed "The Project Peacock".
```

Comment:
I think it should be either:

- Project Peacock (no "the")
- The Peacock Project

I would vote for the first.